src/invwarpedview.jl

```diff
@@ -0,0 +1,118 @@
+"""
+    InvWarpedView(img, tinv, [indices]) -> wv
+
+Create a view of `img` that lazily transforms any given index `I`
+passed to `wv[I]` to correspond to `img[inv(tinv)(I)]`. While
+technically this approach is known as backward mode warping,
+`InvWarpedView` is created using the forward transformation.
```

Comment:
My personal justification for using `tinv` is that it is the inverse of the transformation that is actually requested by the algorithm (which is backward mode warping). I agree it doesn't look nice, but at least it gives it some consistency

If you feel like `InvWarpedView` might be a bit too far out there for the context of this package I could also just move it to the package where I actually need it,